client/src/backend/routes.js

```diff
@@ -514,6 +521,29 @@ const routes = {
             }
           ]
         },
+        {
+          name: "backendRecordsExportTargets",
+          exact: false,
+          component: "ExportTargetsList",
+          path: "/backend/records/export-targets",
+          helper: () => "/backend/records/export-targets",
+          routes: [
+            {
+              name: "backendRecordsExportTargetsNew",
+              exact: true,
+              component: "ExportTargetsNew",
+              path: "/backend/records/export-targets/new",
+              helper: () => "/backend/records/export-targets/new"
+            },
+            {
+              name: "backendRecordsExportTargetEdit",
+              exact: true,
+              component: "ExportTargetsEdit",
+              path: "/backend/records/export-targets/:id",
```

Comment:
Some of these paths include an `/edit` at the end (eg. `"/backend/projects/:id/texts/ingestion/:ingestionId/edit"`) but a majority of them do not. I went with what was most common and left off the `/edit` at the end of the path. Is that alright?

Reply:
I think it's fine for now, although I think it's better to include /edit, since it leaves the other path free for a show or detail view in the future should we need it.